rasa/core/processor.py

```diff
@@ -208,18 +209,22 @@ async def run_anonymization_pipeline(self, tracker: DialogueStateTracker) -> Non
         Args:
             tracker: A tracker representing a conversation state.
         """
-        if self.anonymization_pipeline is None:
+        anonymization_pipeline = plugin_manager().hook.get_anonymization_pipeline()
```

Comment:
To double-check, if we don't have a unit test for this method, could you please add for both when pipeline is None and when it's mocked? 🙏🏻